sdkgen/nodejs-request/lib/index.js

```diff
@@ -1,21 +1,23 @@
 // TODO add options and fetch options
-const getCodegenOptions = require('postman-code-generators').getOptions,
-  processCollection = require('./util').processCollection,
-  sanitize = require('./util').sanitize;
+const processCollection = require('./util').processCollection,
+  sanitize = require('./util').sanitize,
+  combineVariablesLists = require('../../../lib/utils').combineVariablesLists;
 
 /**
  * Generates sdk for nodejs-request
 
  * @param {PostmanCollection} collection - Postman collection Instance
+ * @param {PostmamVariableList} variables - Postman Variable List with resolved scope
  * @param {Object} options - postman-code-generators options
```

Comment:
This generate function should not take p-c-g options. 
You are the consumer of that module, you will decide what options has to be used.